<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateSetScreen</class>
 <widget class="QWidget" name="CreateSetScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="studySetsButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: white;
	border: 2px solid black;
	border-radius: 10px;
}

QPushButton:hover {
	background-color: black;
	color: white;
}</string>
   </property>
   <property name="text">
    <string>Return to Study Sets</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SetName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QObject {
	border: 2px solid black;
}

        QLineEdit
        {
          font: 30px;
        }
      </string>
   </property>
   <property name="placeholderText">
    <string>Enter study set name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createSetButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>520</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS PGothic</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: white;
        width: 200px;
        height: 60px;
        border: 3px solid black;
        border-radius: 10px;
      }

      QPushButton:hover {
          background-color:black;
          color: white;
      }</string>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addQuestionButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>170</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS PGothic</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: white;
        border: 2px solid black;
        border-radius: 5px;
      }

      QPushButton:hover {
          background-color:black;
          color: white;
      }</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeQuestionButton">
   <property name="geometry">
    <rect>
     <x>785</x>
     <y>170</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS PGothic</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: white;
        border: 2px solid black;
        border-radius: 5px;
      }

      QPushButton:hover {
          background-color:black;
          color: white;
      }</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="studyQuestions">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>800</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
	border: 2px solid black;
	margin-top: 2px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/background.png</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>studySetsButton</zorder>
  <zorder>SetName</zorder>
  <zorder>createSetButton</zorder>
  <zorder>addQuestionButton</zorder>
  <zorder>removeQuestionButton</zorder>
  <zorder>studyQuestions</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
